From 1822e8d9c6487a4d99d97c6efeb1708e0bfa4485 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sat, 3 Aug 2013 13:03:40 -0400
Subject: [PATCH] demo: Fix srcdir != builddir, and use libtool for linking

libtool knows how to handle the .la file, don't use -L manually.
---
 demo/Makefile.am |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/demo/Makefile.am b/demo/Makefile.am
index 2f55681..acb6b15 100644
--- a/demo/Makefile.am
+++ b/demo/Makefile.am
@@ -2,8 +2,8 @@ noinst_PROGRAMS = where-am-i
 
 where_am_i_SOURCES = where-am-i.c
 where_am_i_CFLAGS = -I$(top_srcdir)/src 		\
+                    -I$(top_builddir)/src 		\
 		    $(GEOCLUE_CFLAGS)   		\
 		    -DLOCALEDIR="\"$(datadir)/locale\""
-where_am_i_LDADD = -L$(top_srcdir)/src \
-		   -lgeoclue-1.0       \
+where_am_i_LDADD = $(top_builddir)/src/libgeoclue-1.0.la \
 		   $(GEOCLUE_LIBS)
-- 
1.7.1

